src/components/Hero/HubHero/index.tsx

```diff
@@ -8,33 +7,30 @@ import {
   useColorModeValue,
 } from "@chakra-ui/react"
 
-import GatsbyImage from "../../GatsbyImage"
-import { CallToAction } from "../CallToAction"
-import { CommonHeroProps } from "../utils"
+import type { CommonHeroProps } from "@/lib/types"
 
-export interface HubHeroProps extends CommonHeroProps {}
-
-const HubHero = (props: HubHeroProps) => {
-  const { heroImgSrc, title, header, description, buttons } = props
-
-  if (buttons && buttons.length > 2) {
-    throw Error(
-      "Can not have more than two call-to-action buttons in this hero component."
```

Comment:
Why are we removing this error?

Reply:
@pettinarip The type should restrict it to no more than two items, so it would not compile if there are more. In `types.ts` we define the props as:

```ts
export type CommonHeroProps = {
  heroImgSrc: string
  header: string
  title: string
  description: string
  buttons?: [CallToActionProps, CallToActionProps?]
}
```